module/Organizations/src/Organizations/Repository/Filter/PaginationQuery.php

```diff
@@ -62,21 +75,63 @@ public function createQuery($params, $queryBuilder)
             $value = $params;
         }
 
-        if (isset($params['q']) && $params['q']) {
+        if (isset($params['q']) && $params['q'] && $params['q'] != 'en/organizations/profile' ) {
             $queryBuilder->text($params['q'])->language('none');
         }
-        
+
         /*
          * We only show organizations to which the user has view permissions.
          */
-        $queryBuilder->field('permissions.view')->equals($userID);
+        $queryBuilder->field('permissions.view')->in([$userID]);
```

Comment:
In case of organization profile listing, all organizations must be listed. 
Otherwise, guests will not see any organizations on the profile listing.